graphics/hugin-app/Portfile

```diff
@@ -10,7 +10,7 @@ PortGroup               boost 1.0
 
 name                    hugin-app
 version                 2019.2.0
-revision                4
+revision                5
```

Comment:
If the previous revision was not building, then a rev-bump is not really necessary, so can you put this back to 4, thanks.

Reply:
Done